coffee-chats/src/test/java/com/google/step/coffee/data/CalendarUtilsTest.java

```diff
@@ -94,6 +82,15 @@ public void addingAnEventRetriesIfFails() throws IOException {
     verify(insertReturn, times(1)).execute();
   }
 
+  private void setUpEventInfo(List<String> participantIds, List<String> commonTags) {
+    participantIds.add(testUser1Id);
+    participantIds.add(testUser2Id);
+    participantIds.add(testUser3Id);
+
+    commonTags.add("Football");
```

Comment:
In this case it'll be better to put this directly to the test, not to some other function.

I can't point here to the internal doc about unit test best practices, will try to find a better reference